### PR TITLE
"No comments" message, avatar in comments, classic theme improvements, some other changes, Old Navigation

### PR DESCRIPTION
**Pull request by Kiloutre**

### CODE REVIEW

File: models/user.go [hunk 1]
```diff
@@ -45,6 +45,7 @@ type User struct {
 	Language       string    `gorm:"column:language"`
 	Theme          string    `gorm:"column:theme"`
 	AltColors      string    `gorm:"column:alt_colors"`
+	OldNav      string    `gorm:"column:old_nav"`
```
Comment (sfan5) [inline models/user.go:48]: alignment reeee

File: public/css/classic.css [hunk 2]
```diff
@@ -182,15 +186,101 @@ select.form-input {
...
+}
+
+.comment-index a::before {
+	content: '';
+}
+.comment-index a {
+	font-weight: bold;
+	color: black;
+    float: right;
+    margin-left: 9px;
```
Comment (sfan5) [inline public/css/classic.css:262]: indentation reeeee

File: public/css/classic.css [hunk 3]
```diff
@@ -182,15 +186,101 @@ select.form-input {
...
+.comment-submit .comment-form {
+	margin: 0;
+}
+.comment-form h3 {
+	margin-top: 20px;
+}
+
+.comment-submit textarea{
+    height: auto!important;
+    width: auto!important;
```
Comment (sfan5) [inline public/css/classic.css:283]: isn't `!important` supposed to have whitespace in front?

**Review by sfan5 — DISMISSED** [review]

File: public/css/classic.css (hunk 3 above)
Comment (Kiloutre) [reply public/css/classic.css:283]: no

**Review by sfan5 — APPROVED** [review]

**Pull request merged**